pulsar-client-api/src/main/java/org/apache/pulsar/client/api/ClientBuilder.java

```diff
@@ -354,4 +354,26 @@ ClientBuilder authentication(String authPluginClassName, Map<String, String> aut
      * @return the client builder instance
      */
     ClientBuilder connectionTimeout(int duration, TimeUnit unit);
+    
+    /**
+     * Set the duration of time for a backoff interval.
+     * 
+     * @param duration 
+     * 			 the duration of the interval
+     * @param unit
+     * 			 the time unit in which the duration is defined
+     * @return the client builder instance
+     */
+    ClientBuilder defaultBackoffInterval(long duration, TimeUnit unit);
```

Comment:
the backoff implementation is with an exponential interval, so this would be the `startingBackoffInterval()` rather than the `defaultBackoffInterval()`